scripts/error_code_generator_defs/result_codes.json

```diff
@@ -652,6 +652,14 @@
               {
                 "name": "ADUC_ERC_CONTENT_DOWNLOADER_UNSUPPORTED_CONTRACT_VERSION",
                 "value": 13
+              },
+              {
+                "name": "ADUC_ERC_CONTENT_DOWNLOADER_ONDOWNLOADPROC_NOTIMP",
```

Comment:
ERC? Seems inconsistent, but it also looks like you're following what was in this file.